### DISCUSSION TITLE
Windows 11 application frame issue

### DISCUSSION THREAD

**Discussion by the1mason** [AvaloniaUI/Avalonia#9418] [Q&A] [answered]
**Describe the bug**
Application window loses 5 pixels from top and left sides when getting maximized, having system decorations turned off.
Lost pixels become transparent.

When application in normal state, it looks just as expected.

**To Reproduce**

0. Install windows 11 (that is only win11 issue. maybe bc of rounded edges or new layouts feature)

1. Create avalonia .net 5.0 application and go to default (or created one) window.

2. Change window's properties to:
SystemDecorations="None";
WindowState="Maximized";
p.s. also i have TransparencyLevelHint="Blur" and Background="Transparent"
and my window has border as main contaiter. 

3. Run application and open window

**Expected behavior**
Window will be maximized as default and missing +-5 pixels from top and left sides.

**Screenshots**
Normal window state. As you can see, the window has green border from each side.
![image](https://user-images.githubusercontent.com/63291071/136921573-c5688184-b790-498d-9b55-c110cb9156da.png)

Maximized windos (pixels lost)
![image](https://user-images.githubusercontent.com/63291071/136922038-1ea6216c-37a9-491a-afce-8c3bb3752521.png)

**Desktop (please complete the following information):**

- OS: Windows 11
- Version: **Pro Insider Preview Build 22471.rs_prerelease.21.09.29-1415**
also tested on clean windows 11 pro (not preview one) but can't tell build version.

**Comment by grokys** [comment, marked as answer]
I think this is standard win32 behavior that applies to all Windows on that OS:

https://devblogs.microsoft.com/oldnewthing/20150304-00/?p=44543

Here's a SO question relating to the problem with WPF:

https://stackoverflow.com/questions/2967218/window-out-of-the-screen-when-maximized-using-wpf-shell-integration-library/2975574

**Comment by BryanWilkins** [comment]
Just found out by playing a bit after reading the stack overflow articles referenced.  If I put CanResize="false" when the window maximized, this issue does not happen...
